src/azure-cli/azure/cli/command_modules/resource/_validators.py

```diff
@@ -41,6 +41,13 @@ def _validate_template_spec_out(namespace):
     if namespace.output_folder and not os.path.isdir(namespace.output_folder):
         raise CLIError('Please enter a valid output folder')
 
+def validate_deployment_stack_files(namespace):
```

Comment:
> ERROR: /mnt/vss/_work/1/s/src/azure-cli/azure/cli/command_modules/resource/_validators.py:44:1: E302 expected 2 blank lines, found 1
/mnt/vss/_work/1/s/src/azure-cli/azure/cli/command_modules/resource/_validators.py:44:1: E302 expected 2 blank lines, found 1
2     E302 expected 2 blank lines, found 1

For the CLI style issue, you need to add 2 blank lines before the new methods
```suggestion

def validate_deployment_stack_files(namespace):
```